Sources/Logto/LogtoUtilities.swift

```diff
@@ -8,6 +8,13 @@
 import CommonCrypto
 import Foundation
 
+public enum LogtoErrors {
```

Comment:
Do we need to extract the LogtoErrors into an independent file?

Reply:
yes, i did this in the next commit, but reverted 😂